WalletWasabi.Gui/Controls/LockScreen/PinLockScreenViewModel.cs

```diff
@@ -43,10 +43,13 @@ public PinLockScreenViewModel(LockScreenViewModel lockScreenViewModel)
 
 			KeyPadCommand = ReactiveCommand.Create<string>((arg) =>
 			{
-				if (arg == "BACK" && PinInput.Length > 0)
+				if (arg == "BACK")
```

Comment:
@molnard I wonder if replacing the `&&` to `&` to force evaulation of the second condition would work instead of this?

Reply:
The difference is not about the condition but to avoid other else branches to run, that caused the problem. Is it easier to read and add more conditions later.

Reply:
Makes sense, thanks for the insight :)